.github/workflows/main.yml

```diff
@@ -31,7 +33,7 @@ jobs:
         uses: actions/checkout@v1
       - uses: actions-rs/toolchain@v1
         with:
-          toolchain: 1.55.0
+          toolchain: 1.58.0
```

Comment:
Please, make sure to update the toolchain everywhere

Reply:
the rest of the code is using 1.58, this is sync up